rclpy/test/test_qos.py

```diff
@@ -124,3 +126,52 @@ def test_preset_profiles(self):
         assert (
             QoSPresetProfiles.SYSTEM_DEFAULT.value ==
             QoSPresetProfiles.get_from_short_key('system_default'))
+
+
+class TestCheckQosCompatibility(unittest.TestCase):
+
+    def test_compatible(self):
+        qos = QoSProfile(
+            depth=1,
+            durability=QoSDurabilityPolicy.VOLATILE,
```

Comment:
nit: we've explicitly set every policy except for `reliability`. Consider adding it here for completeness.